movie_api/asgi.py

```diff
@@ -0,0 +1,16 @@
+"""
+ASGI config for move_api project.
+
+It exposes the ASGI callable as a module-level variable named ``application``.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/4.0/howto/deployment/asgi/
+"""
+
+import os
+
+from django.core.asgi import get_asgi_application
+
+os.environ.setdefault("DJANGO_SETTINGS_MODULE", "move_api.settings")
```

Comment:
There is a typo in the settings module path. It should be 'movie_api.settings' instead of 'move_api.settings'.